src/main/webapp/app/student/editStudent.xhtml

```diff
@@ -0,0 +1,53 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:h="http://java.sun.com/jsf/html"
+	xmlns:f="http://java.sun.com/jsf/core">
+<h:head>
+	<title>Modifica</title>
+</h:head>
+<h:body>
+	<h:form id="formChangeStudent">
+		<h3>Change Student</h3>
+		<h:panelGrid id="panelChangeStudent" columns="3" border="0">
+			<h:outputLabel for="modificaId" value="#{msg['id']}"/>
+			<h:inputText id="modificaId" value="#{studentController.selectedStudent.id}" disabled="true" />
+			<h:outputText/>
+			
+			<h:outputLabel for="modificaName" value="#{msg['name']}*"/>
+			<h:inputText id="modificaName" value="#{studentController.selectedStudent.name}" required="true" requiredMessage="Name is required"/>
+			<h:message for="modificaName" style="color:red" />
+			
+			<h:outputLabel for="modificaSurname" value="#{msg['surname']}*"/>
+			<h:inputText id="modificaSurname" value="#{studentController.selectedStudent.surname}" required="true" requiredMessage="Surname is required"/>
+			<h:message for="modificaSurname" style="color:red" />
+			
+			<h:outputLabel for="modificaEmail" value="#{msg['email']}"/>
+			<h:inputText id="modificaEmail" value="#{studentController.selectedStudent.email}">
+				<f:validator validatorId="emailValidator" />
+			</h:inputText>	
+			<h:message for="modificaEmail" style="color:red" />
+			
+			<h:outputLabel for="modificaJobTitle" value="#{msg['jobTitle']}"/>
+			<h:inputText id="modificaJobTitle" value="#{studentController.selectedStudent.jobTitle}" />
+			<h:outputText/>
+			
+			<h:outputLabel for="modificaPaymentType" value="#{msg['paymentType']}"/>
+			<h:selectOneMenu id="modificaPaymentType" value="#{studentController.selectedStudent.paymentType}">
+				<f:selectItem />
+				<f:selectItems value="#{studentController.getEnumValues()}" var="item" itemValue="#{item}" itemLabel="#{item.label}" />
+			</h:selectOneMenu>
+			<h:outputText/>
+			
+			<h:outputLabel for="modificaSex" value="#{msg['sex']}*"/>
+			<h:selectOneRadio id="modificaSex" value="#{studentController.selectedStudent.sex}" required="true" requiredMessage="Sex is required">
+				<f:selectItem itemValue="Maschio" itemLabel="M"/>
+				<f:selectItem itemValue="Femmina" itemLabel="F"/>
+			</h:selectOneRadio>
+			<h:message for="modificaSex" style="color:red" />
+			
+			<h:commandButton id="change-button" value="Save Changes" action="#{studentController.updateStudent()}"/>
```

Comment:
All strings should be in the properties file.

Reply:
tutte le stringhe di ogni campo?

Reply:
tutte le  parole dell'applicazioni. Tutti i labels dei bottoni, headers, nome campi etc.  Lo scopo è proprio quello, di avere tutte le parole in una file in modo che si possano cambiare facilmente domani se si vuole gestire una nuova lingua.